encoder/src/test/java/openlr/encoder/routesearch/SecondShortestRouteCheckerTest.java

```diff
@@ -0,0 +1,38 @@
+package openlr.encoder.routesearch;
+
+import openlr.encoder.OpenLREncoderProcessingException;
+import openlr.map.Line;
+import openlr.map.teststubs.OpenLRMapDatabaseAdaptor;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.List;
+
+public class SecondShortestRouteCheckerTest {
+
+    private OpenLRMapDatabaseAdaptor map;
+
+
+   @BeforeTest
+   public void loadMapStub(){
+       InputStream mapFile = OpenLRMapDatabaseAdaptor.class.getClassLoader().getResourceAsStream("teststubs/TestMapStub.xml");
+       this.map = OpenLRMapDatabaseAdaptor.from(mapFile);
+   }
+
```

Comment:
please format you code correctly using the code formatter and remove all unnecessary empty lines.

Reply:
ac5407dc43f6f89b768b92e9571ed9f04942a5d2

Reply:
edafae3f17e96589fda1e8368ec15ddaff1f9fe6